wikidata.mediawiki

```diff
@@ -247,41 +247,42 @@ user_actions:
           label: Message
           type: textarea
           tooltip: the body of the message
+          default: {{subst:Welcome}} --~~~~
       operations:
         - #Appends the message to the user's talk page
           type: append
           page_name: "User_talk:%(user_name)s"
-          markup: "{{subst:Wikipedia:Snuggle/User_message|header=%(header)s|message=%(message)s}}"
+          markup: "{{subst:Wikidata:Snuggle/User_message|header=%(header)s|message=%(message)s}}"
           comment: "New message: %(header)s"
     invite to teahouse:
       label: invite to teahouse
       description: >
-        Posts an invitation to the [[WP:Teahouse]] on 
+        Posts an invitation to the [[WD:Project chat]] on 
         [[User:%(user_name)s|%(user_name)s]]'s 
         [[User_talk:%(user_name)s|talk page]].
-      summary: "%(snuggler)s invited %(user)s to the teahouse"
+      summary: "%(snuggler)s invited %(user)s to the project chat"
       fields:
         - # template
           name: template
           label: Template
           type: radio
           tooltip: the template to be used for the invitation.
-          default: :Wikipedia:Teahouse/Invitation
+          default: :Wikidata:Teahouse/Invitation
```

Comment:
Looks like wikidata doesn't have a Teahouse, so this wouldn't work.  Is there some other newcomer help space we could direct people to?